WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -3271,7 +3272,7 @@ public void populateAnimalRequestTableLookups() throws IOException, CommandExcep
 
     }
 
-    public WebElement fillAnInputByName(String name, String value)
+    private WebElement fillAnInputByName(String name, String value)
```

Comment:
Can this easily be consolidated with the TNPRC copy of the method?

Reply:
Yes, I refactored this function to the ehr helper.